config/config.js

```diff
@@ -88,6 +88,28 @@ var validateSecureMode = function (config) {
 };
 
 /**
+ * Validate Session Secret parameter is not set to default in production
+ */
+var validateSessionSecret = function (config, testing) {
```

Comment:
@ilanbiala that entire file is handled with variable functions like that: https://github.com/jloveland/mean/blob/local-session-secret/config/config.js , the validateSessionSecret is not an exception.